.github/workflows/qt-xml.yml

```diff
@@ -0,0 +1,29 @@
+name: UI XML Format Check
+
+on:
+  push:
+    paths-ignore:
+      - "cmake/**"
+  pull_request:
+    paths:
+      - "UI/forms/**"
+
+jobs:
+  ubuntu64:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Install xmllint
+        run: |
+          sudo apt-get install --no-install-recommends -y libxml2-utils
+
+      - name: Check the Formatting
+        run: |
+          cd UI/forms
+          for i in *.ui; \
+            do xmllint --schema XML-Schema-Qt5.15.xsd --noout "$i"; \
+          done
```

Comment:
Originally I intended to use this in combination with https://github.com/marketplace/actions/xmllint-problem-matcher

It's supposed to annotate in the diff, but ended up not doing so - therefore I've removed that for now.

TODO: Recursive .ui lookup